MAINTAINERS

```diff
@@ -138,10 +138,10 @@ W: http://plugins.geany.org/geanyprj.html
 S: Odd Fixes
 
 geanypy
-P: Lex Trotman <elextr@gmail.com>
-M: Lex Trotman <elextr@gmail.com>
```

Comment:
> The question is what to do when someone makes a pull request to the plugin in GP - one option is to merge it in GP and apply it also upstream, the other is to reject the patch in GP and ask for submitting it upstream.

Unless the GP maintainer intends to have a proper fork, IMO the proper way is to only accept Git-formatted patches from upstream, keeping the authorship and commit messages at least, if possible. Shouldn't allow changes downstream which are not first applied upstream. Something like #440 would enforce this and makes it easier to keep downstream tracking (whatever stable) upstream they want.

> That's real pity it became a real fork - any chance to get the implementations merged so there's just one geanypy?

I don't think it would be really hard to bring them inline, but it would be better to do it after something like #440 so it can't happen again, IMO.

> Which one contains what - the GP one contains the proxy plugin stuff and yours not or the other way round?

Yeah, the main repo doesn't have the proxy plugins stuff merged, mostly because a) it got rebased after I started reviewing/testing b) I wanted to retain compatibility so everybody's plugins didn't break and c) after I gave in and gave @kugel- the OK to merge it, I think he lost interest since it was merged in the fork already.